tda/streaming.py

```diff
@@ -98,7 +115,8 @@ def __init__(self, client, *, account_id=None, enforce_enums=True, ssl_context=N
 
         # Internal fields
         self._request_id = 0
-        self._handlers = defaultdict(list)
+        # Construct a dict for all services
+        self._handlers = dict([(svc, []) for svc in list(Service)])
```

Comment:
Why not use a dict comprehension? Also, why not use a set instead of a list? Assuming we dont want to allow the same handler to be called multiple times with the same message

Reply:
RE: why use a list, to not break the current behaviour (you can currently register a handler multiple times)

Reply:
Sure, but I'm not sure we should support that behavior. :P

@alexgolec?